toolchain/build-toolchain.sh

```diff
@@ -30,76 +45,151 @@ GCC=gcc-9.3.0
 TARGET="$1"
 BUILD_ROOT="$2"
 TOOLCHAIN_PREFIX="${BUILD_ROOT}/${TARGET}"
-MAKEOPTS="-j9 -s"
+MAKEFLAGS="-j9 -s"
```

Comment:
Why `-j9` and not e.g. `-j {numbers of processors}`

Reply:
Common practice to use NPROC+1